contracts/near/eth-types/Cargo.toml

```
@@ -15,11 +15,11 @@ rlp = "0.4.2"
 rlp-derive = "0.1.0"
 ethereum-types = "0.9.2"
 derive_more = "^0.99.2"
-eth2_ssz = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b9865a371ae1d830aa7af1052da09f1c7d0384af" }
-eth2_ssz_derive = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b9865a371ae1d830aa7af1052da09f1c7d0384af" }
-tree_hash = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b9865a371ae1d830aa7af1052da09f1c7d0384af" }
-tree_hash_derive = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b9865a371ae1d830aa7af1052da09f1c7d0384af" }
-eth2_serde_utils = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b9865a371ae1d830aa7af1052da09f1c7d0384af" }
+eth2_ssz = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b01ab64718ecbebb4bb59395d1332980804a0db8" }
+eth2_ssz_derive = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b01ab64718ecbebb4bb59395d1332980804a0db8" }
+tree_hash = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b01ab64718ecbebb4bb59395d1332980804a0db8" }
+tree_hash_derive = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b01ab64718ecbebb4bb59395d1332980804a0db8" }
+eth2_serde_utils = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b01ab64718ecbebb4bb59395d1332980804a0db8" }
```

Using the latest hash (`wasm-near-wip` branch)
```suggestion
eth2_ssz = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
eth2_ssz_derive = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
tree_hash = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
tree_hash_derive = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
eth2_serde_utils = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
```